golang/README.md

```diff
@@ -17,6 +17,47 @@ All golang services written by Clever should be on Go 1.5.
 Some 3rd-party tools that we use, such as Heka, are not on Go 1.5.
 
 
+## Style
+
+The style guides used for Go are [Effective Go](http://golang.org/doc/effective_go.html) and the community [Go style guide](https://code.google.com/p/go-wiki/wiki/CodeReviewComments). There are two tools that can be used to detect common mistakes.
+
+* `go fmt` should be run in any directory that contains go files. It will automatically format the code.
+* `golint file.go` should be run for every go file. It will lint the code and return any issues it finds.
+
+Optionally you can have Vim run the linters for you:
+
+```vim
+" highlight Go errors
+let g:syntastic_go_checkers = ['golint', 'govet']
+```
+
+
+#### Recommended setup
+
+* Makefiles: A Go package should have a Makefile that runs "golint" on all files, e.g.
```

Comment:
this setup is deprecated in favor of the golang makefile lib: https://github.com/Clever/dev-handbook/tree/master/make